src/controllers/message.js

```diff
@@ -65,7 +65,9 @@ const getMessageConversation = async (req, res) => {
             { postedByUser: partnerId, receivedByUser: userId },
             { postedByUser: userId, receivedByUser: partnerId },
         ],
-    }).sort({ createdAt: 1 })
+    })
+        .populate('postedByUser', 'username')
```

Comment:
@larasodjati add the same for recievedByUser too and also for getAllmessages too.

Reply:
@Manizha-khorram done, could you please take a look again?